verifast-proofs/check-verifast-proofs.mysh

```diff
@@ -0,0 +1,13 @@
+cd alloc
+  cd collections
+    cd linked_list.rs
+      verifast -rustc_args "--edition 2021 --cfg test" -skip_specless_fns verified/lib.rs
+      refinement-checker --rustc-args "--edition 2021 --cfg test" original/lib.rs verified/lib.rs
+      diff ../../../../library/alloc/src/collections/linked_list.rs original/linked_list.rs
+    cd ..
+    cd linked_list.rs-negative
+      !verifast -rustc_args "--edition 2021 --cfg test" -skip_specless_fns verified/lib.rs
+      !refinement-checker --rustc-args "--edition 2021 --cfg test" original/lib.rs verified/lib.rs
```

Comment:
@remi-delmas-3000 , just to be clear: you don't like the current approach where the negative test cases are expressed as lines in `check-verifast-proofs.mysh` that start with an exclamation mark, which inverts the exit code (zero iff nonzero)?

Reply:
Notice that the VeriFast action is currently *not* failing.

Reply:
I'd like to separate the two CI actions based on intent: the negative tests are effectively sanity checking verifast, while the positive ones are proving the std-lib code, does that make sense ?

Reply:
Yes, that makes sense; thanks! I had a go at it; let me know what you think!

Reply:
Thank you !